Production/crab/config/2017/Graviton.txt

```diff
@@ -0,0 +1,18 @@
+Automatic
```

Comment:
Why this file is a part of the commit? Those are the old Graviton datasets produced with a wrong config. It should be removed.